tasks/script.json

```diff
@@ -10,6 +10,10 @@
     "arguments": {
       "type": "Array[String]",
       "description": "Arguments to pass to the script"
+    },
+    "extension": {
```

Comment:
The other transports seem to create a tmpdir and copy the file with it's original filename. We should probably do that here too.